ts/src/bingx.ts

```diff
@@ -636,9 +637,15 @@ export default class bingx extends Exchange {
          * @param {object} [params] extra parameters specific to the bingx api endpoint
          * @param {string} [params.price] "mark" or "index" for mark price and index price candles
          * @param {int} [params.until] timestamp in ms of the latest candle to fetch
+         * @param {boolean} [params.paginate] default false, when true will automatically paginate by calling this endpoint multiple times
          * @returns {[[int]]} A list of candles ordered as timestamp, open, high, low, close, volume
```

Comment:
@pcriadoperez Yeah  I have the same question, but if you think about it only `paginationDirection` is specific to the dynamic pagination, all the other options apply

Reply:
Even if it may be repetetive I think it's better to be explicit with the parameters.
Or if not maybe we can include a link to the docs that explains the different paginate parameters?
```suggestion
          * @param {boolean} [params.paginate] default false, when true will automatically paginate by calling this endpoint multiple times. See in the docs all the [availble parameters]  (ttps://github.com/ccxt/ccxt/wiki/Manual#Paginationparams)
         * @returns {[[int]]} A list of candles ordered as timestamp, open, high, low, close, volume
```